ccloader/js/ccloader.js

```diff
@@ -113,14 +109,93 @@ export class ModLoader {
 		}
 	}
 
+
+	_sortModByExtension(manifestPath, modFiles, ccmodFiles, packedMods) {
```

Comment:
When I read `_sortModByExtension` I thought it would sort the arrays passed in. Maybe `_pushToMods` or something similar would be more accurate.